README.md

```diff
@@ -373,6 +372,21 @@ todo: add response example
 
 ```
 
+#### user.positions.getList({ txId })
```

Comment:
Why txId here? it's not the same as in the javascript example code

Reply:
Thx, fixed